contents/docs/data-warehouse/setup/index.mdx

```diff
@@ -41,20 +41,22 @@ See an example in our [S3 setup docs](/docs/data-warehouse/setup/s3).
 
 ## Incremental vs full table
 
-On some sources and tables, you can choose the sync method. The options are incremental replication or full table replication. 
+On some sources and tables, you can choose the sync method. The options are incremental append replication or full table replication. 
 
-### Incremental
+### Incremental append
 
-With incremental replication, you only sync new or updated data. This reduces the total number of rows synced and how long it takes to sync. 
+With incremental append replication, you only sync new data. This reduces the total number of rows synced and how long it takes to sync. 
```

Comment:
Maybe add something highlighting where this is useful. `This is useful for data that is immutable such as invoices` or something like that